### ISSUE TITLE
GNUradio on windows

### ISSUE THREAD

**Issue by shachip** [radioML/dataset#20]
I'm trying to run the code with the Anaconda - Python  (windows). It says "module gnuradio is missing". There is no "gnuradio" package available - Windows version at:

https://anaconda.org/ryanvolz/gnuradio

Should I use it with Linux? Any other solution would be appreciated.

**Comment by sarahghuname** [comment]
hi @shachip  , did you manage to run the code ?

**Comment by shachip** [comment]
No.. But I realized that it is Python code generated from GRC file in GNURadio.. So I can edit the parameters in GRC file and run to generate the dataset.

I hope my understanding is right. If not, please correct me.

**Comment by Ahmed-Abisaif** [comment]
hi @shachip, do you have the GRC file, and can you send it to me

**Comment by efriesema** [comment]
In my experience the best way to run GNU Radio on windows is to install radioconda.  

https://wiki.gnuradio.org/index.php/CondaInstall  

It works on Linux and Mac also and has advantages for all three but for windows it makes keeping it together much easier  just open the radioconda prompt and type in 'gnuradio-companion'  and the graphical user interface pops up.   You can open .grc files directly in here and install Out of Tree packages(OOT)
